modules/virt-creating-windows-vm.adoc

```diff
@@ -10,7 +10,7 @@ You can create a Windows virtual machine (VM) by uploading a Windows image to a
 
 .Prerequisites
 
-* You created a Windows installation DVD or USB with the Windows Media Creation Tool. See link:https://www.microsoft.com/en-us/software-download/windows10%20[Create Windows 10 installation media] in the Microsoft documentation.
+* You created a Windows installation DVD or USB with the Windows Media Creation Tool. See link:https://www.microsoft.com/en-us/software-download/windows10[Create Windows 10 installation media] in the Microsoft documentation.
```

Comment:
FYI, this link opens a page entitled, "Download Windows 10 Disk Image (ISO File)". I'm not sure if you want the link text to match that or not.

Reply:
It does if the website detects you are not accessing it from a Windows operating system, in which case cannot use the Windows Media Creation Tool. I don't know if this behavior is new, but the ISO page explains it in the first paragraph.

Reply:
I am personally more worried that Windows 10 will EOL in October, but rewriting the section is beyond the scope of the bug.